content/resources/chapter-8/building-blocks.tsx

```diff
@@ -165,7 +203,7 @@ const javascriptChallengeFour = {
     name: 'verify',
     args: [],
   },
-  defaultCode: `function get_subsidy(height) {
+  defaultCode: `const getSubsidy = (height)=> {
```

Comment:
We should probably change this to your answer

Reply:
The big int?